assemblies/managing-storage-classes.adoc

```diff
@@ -0,0 +1,22 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+[id="managing-storage-classes"]
+= Managing storage classes
+:upstream:
+//:preview:
+
+ifdef::preview[]
+include::../_artifacts/document-attributes-global.adoc[]
+endif::preview[]
+
+{openshift-platform} cluster administrators use storage classes to describe the different types of storage that is available in their cluster. These storage types can represent different quality-of-service levels, backup policies, or other custom policies set by cluster administrators. 
```

Comment:
Is "describe" the right word, or can a more accurate word be used, such as "classify" or "define"? However, it might that "describe" is actually the right word to use here.

Reply:
Kubernetes uses "describe"